### ISSUE TITLE
Error not logged when running a very large area

### ISSUE THREAD

**Issue by j3r3m1** [orbisgis/geoclimate#885]
Some users have tried to run a very large area. It goes up to the end of the calculation but there is no data processed. It seems the data are downloaded but they are not imported (or correctly imported) within H2GIS database. I wonder where is the problem but there is definitely an error that should be logged while GeoClimate goes up to the end but create no results.

```groovy
{
    "description": "Processing OSM data",
    "input": {
     "area"     : 400000,
    "locations": [[50.568329,5.910666,51.936566,8.096945]]
    },
    "output": {
        "folder": "/tmp"
    },
    "parameters": {
        "rsu_indicators": {
            "indicatorUse": [
                "LCZ",
                "TEB"
            ],
            "svfSimplified": true,
            "estimateHeight": true
        },
        "grid_indicators": {
            "x_size": 1000,
	    "y_size": 1000,
	    "rowCol": false,
	    "output" : "asc",
	    "indicators" :["BUILDING_FRACTION", "BUILDING_HEIGHT", "BUILDING_POP",
                               "BUILDING_TYPE_FRACTION", "WATER_FRACTION", "VEGETATION_FRACTION",
                               "ROAD_FRACTION", "IMPERVIOUS_FRACTION", "FREE_EXTERNAL_FACADE_DENSITY",
                               "BUILDING_HEIGHT_WEIGHTED", "BUILDING_SURFACE_DENSITY",
                               "SEA_LAND_FRACTION", "ASPECT_RATIO", "SVF",
                               "HEIGHT_OF_ROUGHNESS_ELEMENTS", "TERRAIN_ROUGHNESS_CLASS",
                               "UTRF_AREA_FRACTION", "UTRF_FLOOR_AREA_FRACTION",
                               "LCZ_PRIMARY"]
         }
    }
}
```

**Comment by ebocher** [comment]
Thanks I will check

**Comment by ebocher** [comment]
I'm on it

**Comment by ebocher** [comment]
400000 this is a crazy area ;-)

**Comment by ebocher** [comment]
The issue seems at another place. Indeed, Aatest case returns 

```
ava.lang.NullPointerException: Cannot invoke method minus() on null object

	at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:44)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at org.orbisgis.geoclimate.geoindicators.RsuIndicators.frontalAreaIndexDistribution(RsuIndicators.groovy:2020)
```

**Comment by ebocher** [comment]
The returned OSM file is too large. The OSM file contains the following message 

```
<remark> runtime error: Query ran out of memory in "query" at line 18. It would need at least 905 MB of RAM to continue. </remark>
```

**Comment by ebocher** [comment]
Fixed with #892
